uber/site_sections/preregistration.py

```diff
@@ -276,9 +276,10 @@ def dealer_confirmation(self, session, id):
 
     def group_members(self, session, id, message=''):
         group = session.group(id)
+        charge = Charge([group, group.leader]) if group.leader else Charge(group, group.leader)
```

Comment:
Oh whoops, I totally meant to delete the second parameter when I was deleting the brackets. Good catch!